src/components/cc-block-detail/cc-block-detail.js

```diff
@@ -0,0 +1,114 @@
+import { css, html, LitElement } from 'lit';
+import { accessibilityStyles } from '../../styles/accessibility.js';
+import { iconRemixArrowUpSLine as iconArrowUp } from '../../assets/cc-remix.icons.js';
+import '../cc-icon/cc-icon.js';
+
+/**
+ * A display component with mostly HTML+CSS and an open/close toggle feature organized with slots to display information such as CLI commands.
+ *
+ * @cssdisplay block
+ *
+ * @slot tab-title - The title of the tab. Try to only use text.
+ * @slot link - A zone dedicated to link, for example to documentation.
+ * @slot content - A zone dedicated to main content.
+ */
+
+export class CcBlockDetail extends LitElement {
+  static get properties() {
+    return {
+      isOpen: { type: Boolean, reflect: true, attribute: 'is-open' },
+    };
+  }
+
+  constructor() {
+    super();
+
+    /** @type {Boolean} Sets the state of the toggle behaviour. */
+    this.isOpen = false;
+  }
+
+  _onClickToggle() {
+    this.isOpen = !this.isOpen;
+  }
+
+  render() {
+    return html`
+      <div class="menu">
+        <button class="tab ${this.isOpen ? 'open' : 'close'}" @click=${this._onClickToggle}>
+          <slot name="tab-title"></slot>
+          <cc-icon .icon="${iconArrowUp}"></cc-icon>
+        </button>
+        <div class="links">
+          <slot name="link"></slot>
+        </div>
+      </div>
+      ${this.isOpen ? html` <div class="content"><slot name="content"></slot></div> ` : null}
+    `;
+  }
+
+  static get styles() {
+    return [
+      accessibilityStyles,
+      // language=CSS
+      css`
+        :host {
+          display: block;
+        }
+
+        .menu {
+          column-gap: 1em;
+          display: flex;
+          justify-content: space-between;
+        }
+
+        .tab {
+          align-items: center;
+          background-color: transparent;
+          border: none;
+          display: flex;
+          font-size: 1em;
+          padding: 0.25em 0.6em 0.35em;
+        }
+
+        .tab:hover {
+          border-radius: var(--cc-border-radius-default, 0.25em);
+        }
+
+        .tab.open {
+          background-color: var(--cc-color-bg-default, #fff);
+          border-radius: var(--cc-border-radius-default, 0.25em) var(--cc-border-radius-default, 0.25em) 0 0;
+          margin-bottom: 0;
+        }
+
+        .tab cc-icon {
+          transition: all 0.3s;
+        }
+
+        .tab.open cc-icon {
+          transform: rotate(-180deg);
+        }
```

Comment:
note: we can rely directly on the reflected `is-open` attribute with this selector instead:

```
:host([is-open="true"]) cc-icon
```